### PR TITLE
[1LP][RFR] Check for order view

### PR DESCRIPTION
**Pull request by rlbabyuk**
Purpose or Intent
=================
Added additional check for submiting orver view
{{pytest: cfme/tests/services/test_provision_stack.py --long-running}}
PRT: Failures are not caused by these changes

### CODE REVIEW

File: cfme/services/service_catalogs/ui.py [hunk 1]
```diff
@@ -176,6 +176,8 @@ def order(self):
     if BZ(1513541, forced_streams=['5.9']).blocks:
         raise NotImplementedError("Service Order is broken - check BZ 1513541")
     view.submit_button.click()
+    if view.is_displayed and view.submit_button.is_displayed:
+        view.submit_button.click()
```
Comment (quarckster) [inline cfme/services/service_catalogs/ui.py]: **QUESTION**
Why are these checks needed?

Comment (rlbabyuk) [reply cfme/services/service_catalogs/ui.py]: While running the whole module, there are some cases in which submit clicked too fast, and we get an error that last field is not filled, but it is. So I tried some check for field value is set and so on, this check will cover if submit is not successful

Comment (quarckster) [reply cfme/services/service_catalogs/ui.py]: Please add this comment to the code.

Comment (rlbabyuk) [reply cfme/services/service_catalogs/ui.py]: Done

**Pull request merged**